docs/api/README.md

```diff
@@ -83,13 +111,31 @@ If you do not care about exact dimensions, rather size class, it is better to us
 
 ```javascript
 // To make use of width and height in our component, we just access width and 
-// height as props:
-const Info = ({ width = 0, height = 0}) => (
-  <Text>{width}pt x {height}pt</Text>
+// height by using WithContainerDimensions helper:
```

Comment:
Should this say: "by rendering as FaCC (Function as Child Component) inside `WithContainerDimensions` component"?